src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -8,6 +8,20 @@
  */
 public abstract class Command {
 
+    private boolean isUndoable;
```

Comment:
may be good for regular commands to not have an undoable field and
something like if instanceof UndoableCommand instead

Reply:
Hmm I'm doing this because if we're not using isUndoable, the UndoableCommand class will technically be nothing different than the Command class (basically the UndoableCommand.java will be just an empty class declaration) so it's kinda weird.

Any ideas?